lib/views/watch_live/watch_live.dart

```diff
@@ -1,6 +1,8 @@
+import 'dart:async';
+
 import 'package:flutter/material.dart';
+import 'package:thc/models/navigation.dart';
 import 'package:thc/models/theme.dart';
-import 'package:thc/views/widgets.dart';
```

Comment:
`'dart:async'` lets us use a `Timer`.

`'package:thc/models/navigation.dart'` allows us to navigate without needing to pass the `BuildContext` or a `MaterialPageRoute`!

`'package:thc/views/widgets.dart'` was for the `FunPlaceholder` widget, so we don't need it anymore.